Kudu.Services/Functions/FunctionController.cs

```diff
@@ -22,6 +25,100 @@ public FunctionController(IFunctionManager manager, ITraceFactory traceFactory)
             _traceFactory = traceFactory;
         }
 
+        [HttpPut]
```

Comment:
Let's not sweat it now. If we think it can be simplified throughout Kudu, we can do a separate pass for it.